hasher-matcher-actioner/hmalib/lambdas/hashing.py

```diff
@@ -0,0 +1,85 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
```

Comment:
Why `hashing.py` instead of `hasher.py`?

Reply:
No reason. Not held to either name.

Reply:
Not blocking: I kind of prefer hasher since its called hasher-matcher-actioner and we already have a matcher.py and an actioner.py (well technically an action_evaluator.py and an action_performer.py)